### DISCUSSION TITLE
Capture pop up window

### DISCUSSION THREAD

**Discussion by Ineed871** [ImageMagick/ImageMagick#4269] [Help]
Does anyone know of a way to capture a pop up window and just the window on a windows 10 machine. The -window function does not work and neither does any variation of x server.

**Comment by snibgo** [comment]
In Windows, we can capture the entire screen like this:
```
magick screenshot: out.png
```
If your pop-up windows is visible, I expect it would be captured. We can't tell IM to just capture one specific window.
